pages/administration/productions/[productionSlug]/bookings/[bookingReference]/index.vue

```diff
@@ -108,6 +139,16 @@ export default defineNuxtComponent({
         ],
         ['Admin Discount', this.rawBooking.adminDiscountPercentage * 100 + '%']
       ];
+    },
+    anyTicketsChecked() {
+      return this.booking.tickets.some((ticket) => ticket.checkedIn);
+    }
+  },
+  methods: {
+    checkedInAtDisplay(ticket) {
+      return ticket.checkedIn
+        ? `${DateTime.fromISO(ticket.checkedInAt).toHTTP()}`
```

Comment:
```suggestion
        ? DateTime.fromISO(ticket.checkedInAt).toHTTP()
```
Only need to do this if you are trying to concatenate strings in a funky way. If you are just returning a single string it can just be done as normal! Equally, if you aren't using a JS function/expression in the string, can just use `'` or `"`. Doesn't really have any impact as far as I'm aware but it's just the way we've done it in this repo so far

Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals